docs/api.rst

```diff
@@ -1204,9 +1204,9 @@ Since it contains the ``films_id`` foreign key, it is possible to embed ``box_of
 Embedding Views
 ---------------
 
-Embedding a view is possible if the view contains columns that have **foreign keys** defined in their source tables.
+Embedding a view is possible if its columns have **foreign keys** defined in their source table, which, in turn, must be clearly present in the ``FROM`` clause of the view definition (without using sub queries).
```

Comment:
I've been thinking of a further simplification. How about if we refer to source tables as the ones in the FROM/JOIN targets? So in `FROM (SELECT * FROM other_table)`, `other_table` wouldn't be a source table for us.

Reply:
So it'd be:
```suggestion
Embedding a view is possible if its columns have **foreign keys** defined in their source table. Source tables are the tables referenced in the ``FROM`` and ``JOIN`` clauses.
```